cli/tests/integrations/test_marathon.py

```diff
@@ -465,6 +466,86 @@ def test_restarting_app():
         assert stderr == b''
 
 
+def test_killing_app():
+    with _zero_instance_app():
+        _start_app('zero-instance-app', 3)
+        watch_all_deployments()
+        task_set_1 = set([task['id']
+                          for task in _list_tasks(3, 'zero-instance-app')])
+        returncode, stdout, stderr = exec_command(
+            ['dcos', 'marathon', 'app', 'kill', 'zero-instance-app'])
+        assert returncode == 0
+        assert stdout.decode().startswith('Killed tasks: ')
+        assert stderr == b''
+        watch_all_deployments()
+        task_set_2 = set([task['id']
+                          for task in _list_tasks(-1, 'zero-instance-app')])
+        assert len(task_set_1.intersection(task_set_2)) == 0
+
+
+def test_killing_scaling_app():
+    with _zero_instance_app():
+        _start_app('zero-instance-app', 3)
+        watch_all_deployments()
+        _list_tasks(3)
+        command = ['dcos',
+                   'marathon',
+                   'app',
+                   'kill',
+                   '--scale',
+                   'zero-instance-app']
+        returncode, stdout, stderr = exec_command(command)
+        assert returncode == 0
+        assert stdout.decode().startswith('Started Deployment: ')
+        assert stdout.decode().find('version') > -1
+        assert stdout.decode().find('deploymentId') > -1
+        assert stderr == b''
+        watch_all_deployments()
+        _list_tasks(0)
+
+
+def test_killing_with_host_app():
+    with _zero_instance_app():
+        _start_app('zero-instance-app', 3)
+        watch_all_deployments()
+        existing_tasks = _list_tasks(3, 'zero-instance-app')
+        task_hosts = set([task['host'] for task in existing_tasks])
+        if len(task_hosts) <= 1:
+            pytest.skip('test needs 2 or more agents to succeed, '
+                        'only {} agents available'.format(len(task_hosts)))
+        assert len(task_hosts) > 1
+        kill_host = list(task_hosts)[0]
+        expected_to_be_killed = set([task['id']
+                                     for task in existing_tasks
+                                     if task['host'] == kill_host])
+        not_to_be_killed = set([task['id']
+                                for task in existing_tasks
+                                if task['host'] != kill_host])
+        assert len(not_to_be_killed) > 0
+        assert len(expected_to_be_killed) > 0
+        command = ['dcos',
+                   'marathon',
+                   'app',
+                   'kill',
+                   '--host',
+                   kill_host,
+                   'zero-instance-app']
```

Comment:
can we put these on the same line (as much as we can with the 80char limit)? it makes it a little easier to read.